From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Tue, 12 Jul 2016 09:20:19 +0200
Subject: [PATCH] virtfs-proxy-helper: make sure sys/xattr.h is included first

Otherwise building with older glibc versions fails with:

In file included from [...]/include/qemu/xattr.h:26:0,
                 from fsdev/virtfs-proxy-helper.c:26:
/usr/include/i386-linux-gnu/sys/xattr.h:32:3: error: expected identifier before numeric constant
make[1]: *** [fsdev/virtfs-proxy-helper.o] Error 1
make[1]: *** Waiting for unfinished jobs....

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 fsdev/virtfs-proxy-helper.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fsdev/virtfs-proxy-helper.c b/fsdev/virtfs-proxy-helper.c
index 54f7ad1c48f0..1f35a300ef89 100644
--- a/fsdev/virtfs-proxy-helper.c
+++ b/fsdev/virtfs-proxy-helper.c
@@ -9,6 +9,7 @@
  * the COPYING file in the top-level directory.
  */
 
+#include "qemu/xattr.h"
 #include "qemu/osdep.h"
 #include <sys/resource.h>
 #include <getopt.h>
@@ -23,7 +24,6 @@
 #endif
 #include "qemu-common.h"
 #include "qemu/sockets.h"
-#include "qemu/xattr.h"
 #include "9p-iov-marshal.h"
 #include "hw/9pfs/9p-proxy.h"
 #include "fsdev/9p-iov-marshal.h"
